src/new/wallet.js

```diff
@@ -875,6 +962,20 @@ class HathorWallet extends EventEmitter {
     return promise;
   }
 
+  changeConnection(connection) {
```

Comment:
What is it used for?

Reply:
This is used when we change the server in the wallet desktop. In this case I can keep the same `HathorWallet` object but change the `Connection` object, stopping the old conn and starting the new one.

Reply:
I'm wondering why wouldn't we create a new object? I mean, so many things might go wrong that I feel we should just create a new `HathorWallet` instance.

Reply:
I think you are right, there is no reason not to create a new object. Will do that.

Reply:
Removed the method, is not needed anymore.